exo/helpers.py

```diff
@@ -15,6 +17,34 @@
  \___/_/\_\___/ 
     """
 
+def is_apple_silicon():
+    return psutil.MACOS and platform.machine() == 'arm64'
+
+def is_intel_mac():
+    return psutil.MACOS and platform.machine() in ['x86_64', 'i386']
+
+def get_system_info():
+    if psutil.MACOS:
+        if is_apple_silicon():
```

Comment:
Why are these separate functions but below you check psutil.LINUX.
Remove the functions and stick to one convention